bin/mops

```diff
@@ -7,5 +7,6 @@ var program = require('commander');
 program
   .version('0.0.0')
   .command('login [name]', 'Store MMS username and api key.')
+  .command('metrics', 'Metrics subcommand')
```

Comment:
This version does not seem to include the corrected helper for topology, nor the alerts exporting.